### ISSUE TITLE
Installation folder does not exist

### ISSUE THREAD

**Issue by Doomsdayrs** [Revolutionary-Games/Thrive-Launcher#246]
**Describe the bug**
During a fresh install, the "Browse Installed Files" button fails to function.

**To Reproduce**
1. Install ThriveLauncher on a fresh installation
2. Click Settings
3. Click "Installed Versions"
4. Click "Browse Installed Files"
5. Notice error dialog "Folder not found"

**Expected behavior**
Folder opens up with no issue

**Screenshots**
![image](https://user-images.githubusercontent.com/38189170/211872929-8a837f3e-0db4-4926-a38f-114d5e0a94e6.png)

**Desktop (please complete the following information):**
 - OS: Fedora
 - Launcher version: 2.0.3

**Comment by hhyyrylainen** [comment]
Presumably that mentioned installed folder doesn't actually exist, right?
So the bug here is that the "browse installed files" button should not be clickable until a Thrive version has been installed.
That can probably be done with a pretty simple check when opening the settings (or when the install location has changed) to see if the target install folder exists or not and set a view model property that disables the button if it doesn't.

**Comment by Doomsdayrs** [comment]
> Presumably that mentioned installed folder doesn't actually exist, right?

Yep

> So the bug here is that the "browse installed files" button should not be clickable until a Thrive version has been installed.

I would disagree, the button should be clickable, the folder should be attempted to be created on launch.

> That can probably be done with a pretty simple check when opening the settings (or when the install location has changed) to see if the target install folder exists or not

And attempt to create it, if creation fails a flag can be made to warn the user.

**Comment by hhyyrylainen** [comment]
>  the folder should be attempted to be created on launch

I disagree. The launcher is used for example for Steam and itch.io seamless mode. The launcher should not create extraneous folders that may never be required by certain users. 

It was even back in v1 that the install folder is only created the first time a Thrive version is attempted to be created.
